app/components/wikia-in-your-lang.js

```diff
@@ -1,10 +1,10 @@
 import { inject as service } from '@ember/service';
 import Component from '@ember/component';
 import { getOwner } from '@ember/application';
+import localStorageConnector from '@wikia/ember-fandom/utils/local-storage-connector';
 import AlertNotificationsMixin from '../mixins/alert-notifications';
 import LanguagesMixin from '../mixins/languages';
 import WikiaInYourLangModel from '../models/wikia-in-your-lang';
-import localStorageConnector from '../utils/local-storage-connector';
```

Comment:
can we remove this util or is it used in other places?

Reply:
There is no `../utils/local-storage-connector`. It worked because design-system has it in its `app/utils` which is merged with the app tree during the build. It's messy, that's why I used the explicit one.